### PR TITLE
update topic_peers when peer_topics is updated

### PR DESCRIPTION
**Pull request by divagant-martian**
Small piece of code where we update peer_topics but not topic_peers

### CODE REVIEW

**Review by AgeManning — APPROVED** [review]
Thanks, looks like this was missed in an earlier revision

**Review by mxinden — COMMENTED** [review]
Could you bump the patch version and include a changelog entry?

**Comment by divagant-martian** [comment]
> Could you bump the patch version and include a changelog entry?

Is the patch version bump required? There is an unreleased entry already in the Changelog

File: protocols/gossipsub/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # 0.33.1 [unreleased]
 
+- Fix bug in internal peer's topics tracking (see [PR 2325])
```
Comment (mxinden) [inline protocols/gossipsub/CHANGELOG.md]:
```suggestion
- Fix bug in internal peer's topics tracking (see [PR 2325]).
```

**Review by mxinden — APPROVED** [review]
> Is the patch version bump required? There is an unreleased entry already in the Changelog

Ah, yes, sorry, not required.

**Pull request merged**